app/app/dashboard/DashboardView.tsx

```diff
@@ -0,0 +1,151 @@
+          const depositValueRuneInUsd = baseToAsset(baseAmount(memberDetail.runeAdded).minus(memberDetail.runeWithdrawn)).div(pool.assetPrice).times(pool.assetPriceUSD)
+          totalAddedValueInUsd = depositValueAssetInUsd.plus(depositValueRuneInUsd)
+          gainInUsd = totalRedeemValueInUsd.minus(totalAddedValueInUsd)
+        } 
+
+        const isDlp = memberDetail.runeAdded !== '0' && memberDetail.assetDeposit !== '0'
+
+        return {
+          assetId: memberDetail.pool,
+          type: memberDetail.pool.includes('/') ? PositionType.SAVER : isDlp ? PositionType.DLP : PositionType.SLP,
+          deposit: {
+            usd: totalAddedValueInUsd.amount().toNumber(),
+          },
+          gain: {
+            usd: gainInUsd.amount().toNumber(),
+            percentage: gainInUsd
+              .div(totalAddedValueInUsd)
+              .times(100)
+              .amount()
+              .toFixed(4)
+          }
+        }
+      })
+      return { pools, positions }
+    }
+  })
+
+  const totalValue = data?.positions?.reduce((total, position) => {
+    return total + position.deposit.usd + position.gain.usd;
+  }, 0);
+
+  const totalGain = data?.positions?.reduce((total, position) => {
+    return total + position.gain.usd;
+  }, 0);
+
+  const titleStyle = 'my-2 md:mb-4 md:mt-0 md:text-2xl font-medium md:mb-6 text-neutral-900 md:text-neutral font-gt-america-ext uppercase'
+  return (
+    <main className="md:mx-16">
+      <div className="flex flex-col md:mb-10">
+        <h2 className={titleStyle}>
+          Dashboard
+        </h2>
+        <div className="grid grid-cols-6 gap-4 md:gap-8">
+          <div className="col-span-6 md:col-span-2">
+            <DashboardHighlightsCard iconPath="icon-total-value-dashboard.svg" title='Total Value' figure={totalValue && addDollarSignAndSuffix(totalValue) || '-'} />
+          </div>
+          <div className="col-span-3 md:col-span-2">
+            <DashboardHighlightsCard iconPath="icon-total-earnings-dashboard.svg" title='Total Earnings' figure={totalGain && addDollarSignAndSuffix(totalGain) || '-'} />
+          </div>
+          <div className="col-span-3 md:col-span-2">
+            <DashboardHighlightsCard iconPath="icon-points-dashboard.svg" title='Points' />
+          </div>
+        </div>
+      </div>
+      <div className="flex flex-col">
+        <h2 className={titleStyle}>
+          Your positions
+        </h2>
+        <div className="w-2/3 text-neutral-800 text-sm font-normal leading-tight mb-7">Manage your active positions and track your earnings.</div>
+        {
+          data?.positions ? 
+            isPending ? 
+              'Loading...' : 
+                <PositionsList 
+                  positions={data?.positions} 
+                  onAdd={(assetId) => {
+                    setSelectedPool(data?.pools?.find(pool => pool.asset === assetId.replace('/', '.'))) 
```

Comment:
what is assetId? shouldn't it be the same as `pool.asset`? wondering why we need to do any string replacement here

Reply:
The replace function is used to find the pool associated with a position in a saver. In THORChain, while a native asset is represented with the notation BSC.BNB, a synthetic asset is represented as BSC/BNB.

However, since we don’t yet have the logic for Add/Remove operations for savers, I believe it’s better to remove this replace to avoid confusion when the time comes to activate the buttons. This is because LP positions and savers will not behave the same way, and keeping this replace might mask the differences, leading to misunderstandings when it's activated.